pykern/pkcli/fmt.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+u"""run black formatter on file/dir
+
+:copyright: Copyright (c) 2013-2018 Bivio Software, Inc.  All Rights Reserved.
```

Comment:
Definitely copied the copyright from the wrong place. :) If this was copied from a RadiaSoft repo, then fix the original to be an RS copyright with the correct dates.

In emacs, we say C-c C-t fill a template so we don't have to copy and paste from other code.